lib/Utils/Backends.pm

```diff
@@ -59,4 +61,11 @@ sub is_remote_backend {
       check_var('BACKEND', 'spvm');
 }
 
+# In some cases we are using a VNC connection provided by the hypervisor that
+# allows access to the ttys same as for accessing any remote libvirt instance
+# but not what we use for s390x-kvm.
+sub has_ttys {
+    return ((get_var('BACKEND', '') !~ /ipmi|s390x|spvm/) || is_svirt_except_s390x);
```

Comment:
@mnowaksuse @Soulofdestiny can you verify this?

Reply:
> @mnowaksuse @Soulofdestiny can you verify this?

verify what exactly? that those backends don't have ttys? I think in our implementation yes

Reply:
*argh* the logic is still wrong. https://openqa.suse.de/tests/2403368#step/keymap_or_locale/2 failed on s390x-kvm. 

So down to basics:

backend | tty | not-tty
---|---|---
qemu | ✓ | ✕
ipmi | ✕ | ✓
s390x *(s390x-zVM)* | ✕ | ✓
svirt_except_s390x | ✓ | ✕
*(svirt s390x (kvm))* | ✕ | ✓
spvm | ✕ | ✓

I think the semantic problem I have is that I can not use `is_svirt_except_s390x` in the logical combination I guess.

so `(get_var('BACKEND', '') !~ /ipmi|s390x|spvm/) && !get_var('S390_ZKVM'))` should work